### PR TITLE
Added endpoint to fetch tags

### PR DESCRIPTION
**Pull request by umbreak**

### CODE REVIEW

File: delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/models/TagFields.scala [hunk 1]
```diff
@@ -8,7 +8,7 @@ import io.circe.generic.extras.semiauto.deriveConfiguredDecoder
 import scala.annotation.nowarn
 
 /**
-  * The tag fields used as input
+  * The tag fields used as input/output on the routes
```
Comment (imsdu) [inline delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/models/TagFields.scala:11]: As it is used both for input and output, we could just rename it Tag

File: delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala [hunk 2]
```diff
@@ -152,22 +151,32 @@ final class ResourcesRoutes(
...
-                          authorizeFor(AclAddress.Project(ref), resourcePermissions.write).apply {
-                            entity(as[TagFields]) { case TagFields(tagRev, tag) =>
-                              operationName(s"$prefixSegment/resources/{org}/{project}/{schema}/{id}/tags") {
-                                emit(resources.tag(id, ref, schemaOpt, tag, tagRev, rev).map(_.void))
+                        (pathPrefix("tags") & pathEndOrSingleSlash) {
+                          operationName(s"$prefixSegment/resources/{org}/{project}/{schema}/{id}/tags") {
+                            concat(
+                              // Fetch a resource tags
+                              get {
+                                authorizeFor(AclAddress.Project(ref), resourcePermissions.read).apply {
```
Comment (imsdu) [inline delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala:165]: We can include the authorize and the rev, label part in the fetch method no ?

**Review by imsdu — CHANGES_REQUESTED** [review]

**Review by bogdanromanx — APPROVED** [review]
this needs to be added to the api changes

**Comment by umbreak** [comment]
This already existed in [1.4.x](https://github.com/BlueBrain/nexus/blob/v1.4.x/delta/src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/TagRoutes.scala#L69). I've tested it myself against staging. It just wasn't documented

**Pull request merged**